### PR TITLE
{cae,math,vis}[intel/2017b] OpenFOAM v5.0, ParaView v5.4.1, SCOTCH v6.0.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

requires ~~#5682~~ (Qt) + ~~#5683~~ (CGAL)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
node2447.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ccae07ed4352ae39e8d5615d8fe1f64b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2087.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/262435ce1e552ade036b34df7071d955 for a full test report.

File: easybuild/easyconfigs/p/ParaView/ParaView-5.4.1-intel-2017b-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s.tar.gz"]
+checksums = ['390d0f5dc66bf432e202a39b1f34193af4bf8aad2355338fa5e2778ea07a80e4']
+
+dependencies = [
+    ('X11', '20171023'),
+    ('Mesa', '17.2.4'),
+    ('libGLU', '9.0.0'),
+    ('Qt', '4.8.7'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.4.1-intel-2017b-mpi.eb]: switch to `Qt5` for paraview?

Comment (boegel) [reply easybuild/easyconfigs/p/ParaView/ParaView-5.4.1-intel-2017b-mpi.eb]: I guess we could, but I'd rather stick to the same Qt for both `ParaView` & `OpenFOAM`...

**Pull request merged**